tests/python-ldap.spec

```diff
@@ -0,0 +1,79 @@
+# workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1806625
+%global debug_package %{nil}
+
+Name: python-ldap
+Version: 3.1.0
+Release: 9%{?dist}
+License: Python
+Summary: An object-oriented API to access LDAP directory servers
+Source0: %{pypi_source}
+
+BuildRequires: pyproject-rpm-macros
+
+### Build Dependencies ###
+BuildRequires: openldap-devel
+BuildRequires: openssl-devel
+BuildRequires: cyrus-sasl-devel
+
+BuildRequires: gcc
+BuildRequires: openldap-servers
+BuildRequires: openldap-clients
+
+
+%description
+This package contains extension modules. Does not contain pyproject.toml. Has multiple files and directories.
+Building this tests:
+- the proper files are installed in the proper places
+- module glob in %%pyproject_save_files (some modules are included, some not)
+- combined manual and generated Buildrequires
+
+
+%package -n     python3-ldap
+Summary:        %{summary}
+
+Requires:  openldap
+%{?python_provide:%python_provide python3-ldap}
+
+%description -n python3-ldap
+%{summary}
+
+
+%prep
+%setup -q -n %{name}-%{version}%{?prerelease}
+
+
+
+%generate_buildrequires
+%pyproject_buildrequires -t
+
+
+%build
+%pyproject_wheel
+
+
+%install
+%pyproject_install
+%pyproject_save_files ldap* *ldap
+
+
+%check
+%tox
+
+# check if the instalation outputs expected files
+test -d "%{buildroot}%{python3_sitearch}/__pycache__/" 
+test -d "%{buildroot}%{python3_sitearch}/python_ldap-%{version}.dist-info/" 
+test -d "%{buildroot}%{python3_sitearch}/ldap/" 
+test -f "%{buildroot}%{python3_sitearch}/ldapurl.py" 
+test -f "%{buildroot}%{python3_sitearch}/ldif.py" 
+test -d "%{buildroot}%{python3_sitearch}/slapdtest" 
+test -n "$(find '%{buildroot}%{python3_sitearch}' -maxdepth 1 -name '_ldap.cpython-*.so' -print -quit)"
+
+# Not supposed to be listed in %{pyproject_files}. Making sure build will fail if they got listed.
```

Comment:
```suggestion
# Not supposed to be listed in %{pyproject_files}. Making sure build will fail if they got listed.
# The removals must come after %%pyproject_save_files so the test is effective
```

Reply:
I will yet go trough the specs.

Reply:
I don't think that comment is correct. `%pyproject_save_files` only reads the `RECORD` file, other files in `%{buildroot}` are not checked.